### ISSUE TITLE
Input lacks label

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1626]
In the modals that appear after clicking the ‘backup’ and ‘download my cryptdrive’ buttons in ‘Settings/CryptDrive’, there is an input field that lacks a label. This means that screen readers cannot provide users with information about the purpose of the input field.
